src/vellum/workflows/runner/runner.py

```diff
@@ -125,7 +126,6 @@ def __init__(
 
         self._dependencies: Dict[Type[BaseNode], Set[Type[BaseNode]]] = defaultdict(set)
         self._state_forks: Set[StateType] = {self._initial_state}
-        self._mocks_by_node_outputs_class = CycleMap(items=node_output_mocks or [], key_by=lambda mock: mock.__class__)
```

Comment:
This `CycleMap` class ended up being pretty short lived, due to the read/write race it exhibited